docs/src/index.md

```diff
@@ -2,8 +2,9 @@
 
 The *Distributions* package provides a large collection of probabilistic distributions and related functions. Particularly, *Distributions* implements:
 
+* Sampling from distributions
 * Moments (*e.g* mean, variance, skewness, and kurtosis), entropy, and other properties
 * Probability density/mass functions (pdf) and their logarithm (logpdf)
 * Moment generating functions and characteristic functions
```

Comment:
This was already in the docs, but it should be "Moment-generating".

Reply:
done thanks